README.md

```diff
@@ -55,6 +55,8 @@ To build after all dependencies are installed:
 1.57 or newer than 1.63 are NOT supported. If your system Boost version is newer, then you will need to manually build
```

Comment:
Please change the Boost supported range to 1.57 - 1.65 as referenced in https://github.com/bitshares/bitshares-core/issues/1045

Reply:
1.65 has been in master. This PR won't update it.